pygmt/tests/test_grdimage.py

```diff
@@ -51,3 +73,18 @@ def test_grdimage_fails():
     fig = Figure()
     with pytest.raises(GMTInvalidInput):
         fig.grdimage(np.arange(20).reshape((4, 5)))
+
+
+@pytest.mark.mpl_image_compare
+def test_grdimage_over_dateline(xrgrid):
```

Comment:
I'm starting to think that this test is exposing some [flaky tests](https://docs.pytest.org/en/stable/flaky.html) in our test suite (see also https://github.com/GenericMappingTools/pygmt/pull/327#issuecomment-534318735). The test passes pretty much all the time (?) when I just run `pytest --verbose --mpl pygmt/tests/test_grdimage.py` or even `pytest --verbose --mpl pygmt/tests/test_grd*.py`, but running `make test` causes the test to fail :confused:

I probably went down the wrong rabbit hole, but I've tried pretty much all of the [pytest plugins to test for flakiness](https://docs.pytest.org/en/stable/flaky.html#plugins) including pytest-xdist and pytest-flakefinder, and they pretty much agree that it's not `test_grdimage_over_dateline` which is the issue, it's one of the tests that precede it, but I'm having trouble tracking it down since the failures are all intermittent and semi-random.